src/Microsoft.DotNet.Wpf/src/System.Xaml/System.Xaml.csproj

```diff
@@ -99,6 +99,8 @@
     
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Security.Permissions" Version="$(SystemSecurityPermissionsPackageVersion)" />
+    <PackageReference Include="System.Security.Permissions" Version="$(SystemSecurityPermissionsPackageVersion)">
+      <PrivateAssets>All</PrivateAssets>
```

Comment:
nit: you can set this using an attribute, or better make an item definition so future additions don't cause this problem.  May be a moot point if you plan to replace this with a different transport package project in near future.